inscriptions/ISic000482.xml

```diff
@@ -183,23 +188,34 @@
       <desc>Add Description Here</desc>
     </graphic>
   </surface>
-
     -->
     </facsimile>
     <text>
         <body>
-            <div type="edition" xml:space="preserve" subtype="unsupplied" xml:lang="la">
-                <ab><!--Greek or Latin (etc.) text here--></ab>
+            <div type="edition" xml:space="preserve" xml:lang="la" resp="#AR">
+                <ab>
+                    <lb n="1"/><supplied reason="lost"><expan><abbr>imp</abbr><ex>eratori</ex></expan></supplied> <supplied reason="lost"><expan><abbr>caes</abbr><ex>ari</ex></expan></supplied> <supplied reason="lost">divi</supplied> <supplied reason="lost">traiani</supplied>
+                    <lb n="2"/><supplied reason="lost">parthi</supplied>ci <g ref="#interpunct">·</g> <expan><abbr>f</abbr><ex>elix</ex></expan> <g ref="#interpunct">·</g> <name>Traiano</name> <name>Ha<supplied reason="lost">dri</supplied>
+                    <lb n="3" break="no"/><supplied reason="lost">an</supplied>o</name> <g ref="#interpunct">·</g> <expan><abbr>Aug</abbr><ex>usto</ex></expan> <expan><abbr>pontif</abbr><ex>ice</ex></expan> <g ref="#interpunct">·</g>  <expan><abbr>max</abbr><ex>imo</ex></expan> <g ref="#interpunct">·</g> <expan><abbr>trib</abbr><ex>unicia</ex></expan> <g ref="#interpunct">·</g>
+                    <lb n="4"/><expan><abbr>pot</abbr><ex>esta</ex></expan> <g ref="#interpunct">·</g> <num value="3">III</num> <g ref="#interpunct">·</g> <expan><abbr>co</abbr><ex>n</ex>s<ex>ule</ex></expan> <num value="3">III</num>
+                    <lb n="5"/><expan><abbr><choice><reg>l</reg><orig>i</orig></choice></abbr><ex>ocus</ex></expan> <expan><abbr>d</abbr><ex>atus</ex></expan> <expan><abbr>d</abbr><ex>ecreto</ex></expan> <expan><abbr>d</abbr><ex>ecurionum</ex></expan>
```

Comment:
unclear for the L, not reg versus orig, so:
`<expan><abbr><unclear>l</unclear></abbr><ex>ocus</ex></expan>`